changes/4479.changed

```diff
@@ -0,0 +1 @@
+Configured `npm` in Docker image to use 30s timeout, pin `npm` to 9.X, and use `npm ci` instead of `npm install` to improve builds.
```

Comment:
```suggestion
Configured `npm` in Docker image to use 30s timeout, pinned `npm` to 9.X, and changed Docker build to use `npm ci` instead of `npm install` to improve builds.
```